From f4a3e62c9ee4208907b8a077a519397a4de26b8e Mon Sep 17 00:00:00 2001
From: "Philip, Avinash" <avinashphilip@ti.com>
Date: Sat, 10 Mar 2012 12:22:28 +0530
Subject: [PATCH 0883/1092] PWM: ehrpwm: Replace __raw [read/write] with
 read/write variant

Replaces __raw_read/__raw_write functions with read/wrte, as this
functions inserts memory barrier instructions required with ARMv6+

Signed-off-by: Philip, Avinash <avinashphilip@ti.com>
---
 drivers/pwm/ehrpwm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/pwm/ehrpwm.c b/drivers/pwm/ehrpwm.c
index c0d3456..b6fa0ad 100644
--- a/drivers/pwm/ehrpwm.c
+++ b/drivers/pwm/ehrpwm.c
@@ -176,13 +176,13 @@ struct ehrpwm_suspend_params {
 static inline unsigned short ehrpwm_read(struct ehrpwm_pwm *ehrpwm,
 	unsigned int offset)
 {
-	return __raw_readw(ehrpwm->mmio_base + offset);
+	return readw(ehrpwm->mmio_base + offset);
 }
 
 static inline void ehrpwm_write(struct ehrpwm_pwm *ehrpwm, unsigned int offset,
 	unsigned short val)
 {
-	__raw_writew(val, ehrpwm->mmio_base + offset);
+	writew(val, ehrpwm->mmio_base + offset);
 }
 
 static void ehrpwm_reg_config(struct ehrpwm_pwm *ehrpwm, unsigned int offset,
-- 
1.7.11.2

